apps/src/lab2/lab2Redux.ts

```diff
@@ -45,6 +45,8 @@ export interface LabState {
   isLoadingProjectOrLevel: boolean;
   // If the lab is loading. Can be updated by lab-specific components.
   isLoading: boolean;
+  // Whether the lab is currently running.
+  isRunning: boolean;
```

Comment:
This makes me a bit nervous 🙂 probably being a little over-cautious because this feels a bit StudioApp-y, but I also don't think it feels right to have Lab2 know about the "running" state of a lab, since that can vary quite a bit from lab to lab (ex. AI Chat vs Music Lab). It feels like that's something the lab should manage internally.

Reply:
Good point.  I also felt uncomfortable about this, especially since it duplicated `MusicState`'s `isPlaying`.